cdap-ui/app/cdap/services/ThemeHelper.ts

```diff
@@ -344,7 +344,6 @@ function parse1Point0Spec(themeJSON: IOnePoint0SpecJSON): IThemeObj {
           'wrangler-datamodel-viewer'
         );
       }
-
```

Comment:
let's avoid unnecessary changes to the CDAP code unless it is recommended by formatting tool.

Reply:
Ok, I will take it into account in the future.